modules/nodes-cluster-resource-levels-command.adoc

```diff
@@ -4,20 +4,22 @@
 
 :_content-type: PROCEDURE
 [id="nodes-cluster-resource-levels-command_{context}"]
-= Running the cluster capacity tool on the command line
+= Running the OpenShift Cluster Capacity Tool on the command line
 
-You can run the {product-title} cluster capacity tool from the command line
+You can run the OpenShift Cluster Capacity Tool from the command line
 to estimate the number of pods that can be scheduled onto your cluster.
 
+You create a sample pod spec file, which the tool uses for estimating resource usage. The pod spec specifies its resource
+requirements as `limits` or `requests`. The cluster capacity tool takes the
+pod's resource requirements into account for its estimation analysis.
```

Comment:
[Existing text moved](https://github.com/openshift/openshift-docs/pull/63085/files#diff-9a6729224de7e14a526bdbeae13a8ce30330160e7bb6bf9a50fa0e7d37374a35L16) out of procedure.